tools/src/CSVFileImport.cc

```diff
@@ -420,13 +421,13 @@ int main(int argc, char* argv[]) {
           case orc::STRING:
           case orc::CHAR:
           case orc::VARCHAR:
-          case orc::BINARY:
+          case orc::BINARY: 
+            bufferList.emplace_back(*orc::getDefaultPool(), 4 * 1024 * 1024);
```

Comment:
Maybe we can reduce the buffer size to 1 * 1024 * 1024 now since it's used per column.
We use 1MB buffer size in column writers:
https://github.com/apache/orc/blob/6f4481565761400587c7a266ca4c0a4244c0aa34/c%2B%2B/src/ColumnWriter.cc#L49-L62

Reply:
Yes, that's reasonable, I had been considering changing the memory allocaton too.